src/observable/ConnectableObservable.ts

```diff
@@ -39,6 +39,13 @@ export class ConnectableObservable<T> extends Observable<T> {
   refCount(): Observable<T> {
     return new RefCountObservable(this);
   }
+
+  // This method is opened for `ConnectableSubscription`.
```

Comment:
it's just curiosity and assume it's not - does this comment appears editor's intellisense for typings? does JSDoc style comment appears?